packages/amplify-category-auth/src/provider-utils/awscloudformation/service-walkthrough-types.ts

```diff
@@ -27,6 +27,7 @@ export interface ServiceQuestionsBaseResult {
   userpoolClientReadAttributes: string[];
   userpoolClientWriteAttributes: string[];
   usernameCaseSensitive?: boolean;
+  useEnabledMfas?: boolean;
```

Comment:
Why is this an optional attribute ?  can it not be required, but set to false by default ?

Reply:
It's because the update flow might not necessarily have that option, since the value is read from FF